source/Xunit.Gherkin.Quick/CoreModel/Scenario/GherkinScenarioOutlineExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace Xunit.Gherkin.Quick
 {
     internal static class GherkinScenarioOutlineExtensions
     {
-        private static readonly Regex _placeholderRegex = new Regex(@"<([^>]+)>");
+        private static readonly Regex _placeholderRegex = new Regex(@"<([a-zA-Z0-9]([^<|^>]*[a-zA-Z0-9])?)>");
```

Comment:
This seems wrong. This part: `[^<|^>]`
You intend to say to match anything except < and >, right? The correct regex for that is `[^<>]`.
What you have now matches everything except < and | and ^ and >. I don't think you mean that. You need only to exclude two symbols.

Reply:
You're right, I'll add tests for this case.